### PR TITLE
PD-826 download url only response added in download methods

### PR DESCRIPTION
**Pull request by mrasif**
* `link_only` filter added in download method.
* This changes required to get download url only, without having file content in response,
  So, download process can be done later.

### CODE REVIEW

File: gpapi/googleplay.py [hunk 1]
```diff
@@ -635,6 +636,9 @@ def delivery(self, packageName, versionCode=None, offerType=1,
             result['additionalData'] = []
             result['splits'] = []
             downloadUrl = response.payload.deliveryResponse.appDeliveryData.downloadUrl
+            result['downloadUrl'] = downloadUrl
+            if link_only:
```
Comment (sureshdsk) [inline gpapi/googleplay.py:640]: this download method works in multiple ways.
1. single apk - which you've implemented
2. split apk files - multiple apk links will be there - missing
3. additional obb file links also there - missing - not sure of its use at the moment. 

so i feel its better to create a new method and do this.

**Review by sureshdsk — CHANGES_REQUESTED** [review]
need changes.

File: gpapi/googleplay.py (hunk 1 above)
Comment (mrasif) [reply gpapi/googleplay.py]: Added specific method for extracting download link

**Comment by subho007** [comment]
Closing this PR since it is irrelevant now

**Pull request closed without merging**